packages/react-storage/src/components/StorageBrowser/tasks/useProcessTasks.ts

```diff
@@ -191,16 +191,12 @@ export const useProcessTasks = <
           onTaskSuccess(task, output?.value);
         }
 
-        if (task && isFunction(onSuccess)) onSuccess(data, output?.value);
```

Comment:
Removing `onSuccess` and `onError` on ln 202 as they never run Related interface changes will be handled in a follow up